landing-page/content/common/vendors.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Vendors"
+url: vendors
+disableSidebar: true
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+## Vendors Supporting Iceberg Tables
```

Comment:
@jasonhughes248, would you like to add a section for Dremio?

Reply:
@rdblue yeah, that'd be great, thanks!

doesn't look like I have the permissions necessary to add it myself, so here's the content:

### [Dremio](https://www.dremio.com/)

Dremio is the easy and open data lakehouse platform. With Dremio, an organization can easily build and manage a data lakehouse in which data is stored in open formats like Apache Iceberg and can be processed with Dremio’s interactive SQL query engine and non-Dremio processing engines. [Dremio Cloud](https://www.dremio.com/get-started/) provides these capabilities in a fully managed offering.

[Dremio Sonar](https://www.dremio.com/platform/sonar/) is a lakehouse query engine that provides interactive performance and DML on Apache Iceberg, as well as other formats and data sources. It gives data providers the ability to quickly provision new views and metrics on Apache Iceberg tables and other data sources without ETL pipelines. It gives data consumers governed self-service and the ability to interact with a logical data model.

[Dremio Arctic](https://www.dremio.com/platform/arctic/) is a lakehouse catalog and optimization service for Apache Iceberg. Arctic makes it easy to manage tables in the lakehouse, and automatically optimizes the data in the background to ensure high-performance access through Dremio Sonar, Spark, Flink, Snowflake, Athena, EMR, and other engines.

Reply:
Added, but I cut out some of the sentences that would appear like the ASF project is making claims about Dremio (as we did for the other entries here). Feel free to suggest edits if you think I got it wrong!

Reply:
ah @rdblue makes sense. yup your changes look good